core/web/cosmos_transfer_controller.go

```diff
@@ -27,11 +27,13 @@ type CosmosTransfersController struct {
 	App chainlink.Application
 }
 
+var ErrCosmosNotEnabled = errChainDisabled{name: "Cosmos", tomlKey: "Cosmos.Enabled"}
```

Comment:
Perhaps we could standardize these around `errors.Unsupported`: https://pkg.go.dev/errors#pkg-variables

Reply:
By implementing `Is` on `errChainDisabled`? I'll leave this as a follow up

Reply:
Yeah better to follow up.

I was imagining we could shed the per family enumeration too, since we'll want to be more dynamic eventually.